modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/TranslationManager.testcase

```diff
@@ -40,6 +40,44 @@ definition {
 		}
 	}
 
+	@description = "LPS-148143. Translations manager has no success message and undo option after adding language to translations list."
+	@priority = "3"
+	test HasNoSuccessMessageAndUndoOptionAfterAddingLanguage {
+		task ("Given manage translations modal") {
+			TranslationManagerPortlet.goToManageTranslationsModal(
+				key_currentLocale = "en-us",
+				key_title = "Admin");
+		}
+
+		task ("When add language in Manage Translations table list") {
+			ManageTranslations.addLanguage(key_locale = "zh-CN");
+		}
+
+		task ("Then assert no success message and undo option") {
+			AssertElementNotPresent(locator1 = "Message#SUCCESS_DISMISSIBLE");
+
+			AssertElementNotPresent(
+				key_text = "undo",
+				locator1 = "Icon#ANY_DISABLED");
+		}
```

Comment:
To be honest, this is a weird assertion that I wouldn't think is needed. I don't think I would care if a success message is not shown. I think the main focus of the test should just be `CanAddLanguageToTranslationList` rather than there isn't a success message shown.

cc/ @Hijky @zhangyan0701

Reply:
Please correct me if I'm wrong or this was discussed before.

cc/ @zhangyan0701 @tinycarol

Reply:
@john-co For this, I wrote the following two together. I think the two test points have no successful message and undo option after adding language and adding respectively. 


Given: Manage Translations modal is open
When: Add a language from the dropdown list
And When: Click Done button
Then: Assert Manage Translations modal is closed
And Then: Click the language selector, assert the language is active 
-- | -- | -- | --
  | Given: Manage Translations modal is open
When: Add a language from the dropdown list
Then: Assert no success message and undo option

Reply:
I don't think the assertion for success message and undo option is necessary. I looked over the epic's acceptance criteria https://issues.liferay.com/browse/LPS-140069 and scanned through the proposal: https://liferay.atlassian.net/wiki/spaces/ENGLEXICON/pages/1895696034/LEXI-1236+Translation+Manager+Review+and+Improvements and there's no reference to this workflow to make sure there's no success message/undo option, and if it did, it doesn't justify it to be included in a level 5 test.

If this assertion failed what does it catch? That there is a success message or undo option. But imo, it doesn't matter if this message appeared or not. It wouldn't cause a functional regression on the Translation Manager. What matters more is that the language is active, and if this failed then it's likely an FP5 bug scenario.

Reply:
OK， I will do it.

Reply:
I create a new PR.